### DISCUSSION TITLE
Affirm catridge Integration Issues

### DISCUSSION THREAD

**Discussion by shihasmandottil** [SalesforceCommerceCloud/sfcc-ci#541] [Team Posts]
Hi,
I am integrating affirm in the checkout process for payment methods.I have created Affirm as a payment method and BASIC_CREDIT as a processor.When i select affirm it redirects to the the Affirm window to create affirm account, selecting Autopay options,Selecting Loan Tenure,etc.After fetching all these documents it redirects back to the storefront to a controller method  called **Affirm-Update** Where it is validating the credit card details. In the entire affirm flow we are not linking users credit card information. But when it reaches back to the controller it is showing credit card number as 4111111111111111 and gives invalid credit card number and hence terminates the flow. Can anyone help me from where the credit card is linked to the Affirm flow.

**Comment by olexandr-zubkov** [comment]
Hi @shihasmandottil 
As I know Affirm is designed for it's own payment processor. When you are trying to use BASIC_CREDIT there can be hardcoded credit card number.
